include/psa/crypto_sizes.h

```diff
@@ -87,19 +91,42 @@
 
 /** \def PSA_HASH_MAX_SIZE
  *
- * Maximum size of a hash.
+ * Maximum output size of a hash.
```

Comment:
The hash is the output, so “maximum size” was correct.

Reply:
reverted